src/clusterfuzz/_internal/issue_management/issue_tracker_utils.py

```diff
@@ -43,8 +44,16 @@ def _get_issue_tracker_project_name(testcase=None):
   return data_handler.get_issue_tracker_name(job_type)
 
 
-@memoize.wrap(memoize.FifoInMemory(256))
-def get_issue_tracker(project_name=None):
+def get_issue_tracker(project_name=None, is_appengine=True):
+  """Get the issue tracker with the given type and name."""
+  if is_appengine:
+    return request_cache.wrap(capacity=_ISSUE_TRACKER_CACHE_CAPACITY)(
+        _get_issue_tracker(project_name))
+  return memoize.wrap(memoize.FifoInMemory(256))(
```

Comment:
nit: you can probably rewrite all of this in a more Pythonic way as a decorator:

```python
def request_or_task_cache(func):
  """Cache that lasts for a bot task's lifetime, or an App Engine request lifetime.""" 
  if environment.is_running_on_app_engine():
    return request_cache.wrap(capacity=...)(func)
  return memoize.wrap(memoize.FifoInMemory(256))(func)

@request_or_task_cache
def get_issue_tracker(...)
```